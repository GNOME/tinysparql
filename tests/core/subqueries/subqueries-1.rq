SELECT ?group ?last_date ?last_message
    (SELECT COUNT(?message) AS ?message_count WHERE { ?message example:group ?group })
WHERE {
    SELECT ?group ?last_date
        (SELECT ?message WHERE { ?message example:group ?group ; example:date ?date } ORDER BY DESC(?date) LIMIT 1) AS ?last_message
    WHERE {
        ?group example:lastDate ?last_date .
    } ORDER BY DESC(?last_date)
}

