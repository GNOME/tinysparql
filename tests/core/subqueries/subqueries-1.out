"http://example.org/ns#group2"	"2009-12-05T00:00:00Z"	"http://example.org/ns#msg5"	"3"
"http://example.org/ns#group1"	"2009-12-02T00:00:00Z"	"http://example.org/ns#msg2"	"2"
